source/algorithms.tex

```diff
@@ -555,13 +555,13 @@
                                            BidirectionalIterator first,
                                            BidirectionalIterator last,
                                            Predicate pred);
-  template <class InputIterator, class OutputIterator1,
+  template<class InputIterator, class OutputIterator1,
             class OutputIterator2, class Predicate>
```

Comment:
This line and those like it need to have their alignment fixed.

Reply:
Agreed. I've scriptified the search and found a few pre-existing misalignments, too. Fixed.